presto-native-execution/presto_cpp/main/SessionProperties.h

```diff
@@ -86,6 +98,8 @@ class SessionProperties {
   /// The maximum allowed spill file size.
   static constexpr const char* kMaxSpillFileSize = "native_max_spill_file_size";
 
+  static constexpr const char* kMaxSpillBytes = "native_max_spill_bytes";
```

Comment:
It might be good to specify all the operators (Hashjoin, HashAgg, OrderBy, RowNumber, Window etc) to which this value applies

Reply:
I don't have much context on spilling so might not have the full list, @xiaoxmeng do you know the full list? 

I can add in a separate PR. We need this change to make an internal release cut.

Reply:
@kewang1024  @aditi-pandit: this applies at task level and is not specific to a particular operator.